store-cli_test.go

```diff
@@ -100,3 +100,127 @@ func TestMakeURL(t *testing.T) {
 		t.Fatalf("Expected error, got nil")
 	}
 }
+
+func TestGetTimeout(t *testing.T) {
+	testCases := []struct {
+		name           string
+		flagTimeout    string
+		envName        string
+		defaultTimeout int
+		expected       int
+		envValue       string
+		shouldError    bool
+	}{
+		{
+			name:           "use flag timeout",
+			flagTimeout:    "50",
+			envName:        "SD_STORE_CLI_DOWNLOAD_HTTP_TIMEOUT",
+			defaultTimeout: 60,
+			expected:       50,
+			envValue:       "",
+			shouldError:    false,
+		},
+		{
+			name:           "use environment timeout",
+			flagTimeout:    "",
+			envName:        "SD_STORE_CLI_DOWNLOAD_HTTP_TIMEOUT",
+			defaultTimeout: 60,
+			expected:       70,
+			envValue:       "70",
+			shouldError:    false,
+		},
+		{
+			name:           "use default upload timeout",
+			flagTimeout:    "",
+			envName:        "SD_STORE_CLI_UPLOAD_HTTP_TIMEOUT",
+			defaultTimeout: UPLOAD_HTTP_TIMEOUT,
+			expected:       60,
+			envValue:       "",
+			shouldError:    false,
+		},
+		{
+			name:           "use default download timeout",
+			flagTimeout:    "",
+			envName:        "SD_STORE_CLI_DOWNLOAD_HTTP_TIMEOUT",
+			defaultTimeout: DOWNLOAD_HTTP_TIMEOUT,
+			expected:       300,
+			envValue:       "",
+			shouldError:    false,
+		},
+		{
+			name:           "use default remove timeout",
+			flagTimeout:    "",
+			envName:        "SD_STORE_CLI_REMOVE_HTTP_TIMEOUT",
+			defaultTimeout: REMOVE_HTTP_TIMEOUT,
+			expected:       300,
+			envValue:       "",
+			shouldError:    false,
+		},
+		{
+			name:           "set flagTimeout to zero",
+			flagTimeout:    "0",
+			envName:        "SD_STORE_CLI_UPLOAD_HTTP_TIMEOUT",
+			defaultTimeout: UPLOAD_HTTP_TIMEOUT,
+			expected:       0,
+			envValue:       "",
+			shouldError:    false,
+		},
+		{
+			name:           "set envValue to zero",
+			flagTimeout:    "",
+			envName:        "SD_STORE_CLI_UPLOAD_HTTP_TIMEOUT",
+			defaultTimeout: UPLOAD_HTTP_TIMEOUT,
+			expected:       0,
+			envValue:       "0",
+			shouldError:    false,
+		},
+		{
+			name:           "Error case set flagTimeout to string",
+			flagTimeout:    "dummystring",
+			envName:        "SD_STORE_CLI_UPLOAD_HTTP_TIMEOUT",
+			defaultTimeout: UPLOAD_HTTP_TIMEOUT,
+			expected:       999,
```

Comment:
I think it's better for 'expected' to be 0, since strconv.Atoi returns 0 when the conversion fails. 
This doesn't affect the result, but having an unrelated 999 as 'expected' could potentially cause confusion.

Reply:
OK. you're right. I will fix it.